packages/presence-signal/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@solid-primitives/presence-signal",
```

Comment:
similarly to the comment above, `presence` should be enough

Reply:
Done 👍